includes/Utilities/Plugin_Request_Utility.php

```diff
@@ -161,4 +161,24 @@ public static function get_directories_to_ignore() {
 
 		return $directories_to_ignore;
 	}
+
+	/**
+	 * Gets the files to ignore using the filter.
+	 *
+	 * @since 1.0.0
```

Comment:
Note that this is incorrect now

Reply:
Should it be `1.0.2` or `1.1.0`? What will be our next release?

Reply:
As we set `1.0.2` for the most recent merged changes, let's stick with that version for these changes as well.